common-healthchecks.yml

```
@@ -0,0 +1,28 @@
+    healthcheck:
+      interval: 10s
+      timeout: 5s
+      retries: 3
+      start_period: 30s
+
+  mockoon:
+    healthcheck:
+      interval: 10s
+      timeout: 5s
+      retries: 3
+      start_period: 30s
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)